source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-This guide shows you how to create an application that uses the Node.js
-driver to connect to a MongoDB Atlas cluster.
+*Estimated time to complete this guide: 35 minutes.*
 
-Check for Node and NPM
-----------------------
+This guide shows you how to create an application that uses the
+**Node.js driver** to connect to a **MongoDB Atlas cluster**.
 
-Ensure your system has Node version 8 or later and NPM (Node Package
-Manager) installed. Run the following commands to check that you meet
-these requirements:
+The Node.js driver is an interface through which you can connect to and
+communicate with MongoDB instances. If you prefer to connect to MongoDB using
+a different programming language, select from the ones on our
+:driver:`drivers documentation page <>`.
 
-First, check the version of Node:
+MongoDB Atlas is a fully-managed cloud database service that hosts your data
+on MongoDB instances. We show you how to get started with your own free (no
+credit card required) instance in this guide.
 
-.. code-block:: sh
+Follow the steps below to connect your Node.js application with a MongoDB
+instance.
 
-   node -v
+Set up Your Project
```

Comment:
Should "up" be "Up" ?

Reply:
By title case standards, it's lowercase because it's a preposition.